internal/evaluator/conftest_evaluator_test.go

```diff
@@ -1572,10 +1572,26 @@ func TestNameScoring(t *testing.T) {
 			name:  "pkg:term",
 			score: 110,
 		},
+		{
+			name:  "path.pkg:term",
+			score: 210,
```

Comment:
Would these scores be different based on the changes to weighting in `conftest_evaluator.go`?

Reply:
It's supposed to not change the behavior for existing names, unless the have multiple elements in the path.